crates/neon-sys/native/src/neon.cc

```diff
@@ -60,15 +60,15 @@ extern "C" void *Neon_Object_GetIsolate(v8::Local<v8::Object> obj) {
   return obj->GetIsolate();
 }
 
-extern "C" void Neon_Primitive_Undefined(v8::Local<v8::Primitive> *out) {
+extern "C" void Neon_Primitive_Undefined(v8::Local<v8::Primitive> *out, v8::Isolate *isolate) {
```

Comment:
:+1: This is great, thanks. We're moving forward treating `neon-sys` as an internal API, therefore, this is the preferred approach to limit branching elsewhere.